<ui version="4.0" >
 <class>frmReport</class>
 <widget class="QDialog" name="frmReport" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>450</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Zgłoś niepasujące napisy</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/resources.qrc" >
    <normaloff>:/icon/qnapi-48-icon.png</normaloff>:/icon/qnapi-48-icon.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="lbMovieSelect" >
     <property name="text" >
      <string>&amp;Wybierz plik z filmem:</string>
     </property>
     <property name="buddy" >
      <cstring>pbMovieSelect</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLineEdit" name="leMovieSelect" />
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="pbMovieSelect" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="../res/resources.qrc" >
       <normaloff>:/ui/folder.png</normaloff>:/ui/folder.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lbLanguage" >
     <property name="text" >
      <string>&amp;Język napisów:</string>
     </property>
     <property name="buddy" >
      <cstring>cbLanguage</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="cbLanguage" >
     <item>
      <property name="text" >
       <string>Polski</string>
      </property>
      <property name="icon" >
       <iconset resource="../res/resources.qrc" >
        <normaloff>:/languages/pl.gif</normaloff>:/languages/pl.gif</iconset>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Angielski</string>
      </property>
      <property name="icon" >
       <iconset resource="../res/resources.qrc" >
        <normaloff>:/languages/en.gif</normaloff>:/languages/en.gif</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="lbProblem" >
     <property name="text" >
      <string>&amp;Problem:</string>
     </property>
     <property name="buddy" >
      <cstring>cbProblem</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QComboBox" name="cbProblem" >
     <item>
      <property name="text" >
       <string>Napisy wyświetlają się w nieodpowiednim momencie</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Napisy są do innego filmu</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Napisy przetłumaczone przez komputerowy translator</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Tylko część napisów</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Inny powód (wpisz poniżej)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0" colspan="3" >
    <widget class="QLineEdit" name="leProblem" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lbAction" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Wskaż plik z filmem oraz opisz problem.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&amp;Anuluj</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbReport" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Wyślij raport</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>frmReport</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>305</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel" >
     <x>423</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
